### ISSUE TITLE
New release metadata screen is ugly

### ISSUE THREAD

**Issue by CodeF53** [modrinth/knossos#820]
### Is your suggested enhancement related to a problem? Please describe.

The new release metadata screen looks garbage. 

Not sure where this is currently active, but I saw it first at the [datapack test](https://knossos-git-datapack-support-modrinth.vercel.app/mod/test-project/version/wdawdwda+mod).

### Describe the solution you'd like

Written by @Obscure2020:
The title bar that also contains the download, back, and report buttons should be shrunk leftwards so it is the same width as the Changelog and Files bubbles. The Metadata bubble should then be moved upwards so that its top aligns with the title bar bubble. Lastly, the title bar bubble, the Changelog bubble, and the Files bubble should be combined into one singular bubble. Make them sections of one bubble, not three separate bubbles.
![image](https://user-images.githubusercontent.com/37855219/209768787-b7744823-aa3d-48e4-be84-e861b258afa5.png)

### Additional context

How it is:
![image](https://user-images.githubusercontent.com/37855219/209768957-1f440885-c32b-4f28-b0cb-815206fbd651.png)

What I think would look better (hacky fast image edit, excuse bad margins):
![image](https://user-images.githubusercontent.com/37855219/209769212-bf3b06f3-1bc1-4e9d-9cd6-c983a6e4caa3.png)

**Comment by Minenash** [comment]
Personally, I still like the separation of action/header, changelog, and files
![image](https://user-images.githubusercontent.com/12193049/209769809-fc3d7970-a017-4598-a9ce-e237a7125202.png)

**Comment by HyperSoop** [comment]
I think metadata could also be reordered to release-versions-loaders-downloads-number top to bottom too, so more relevant information is closer to the top, though that's a bit opinionated (but version number should not be the second one at all!)

**Comment by Minenash** [comment]
I agree with the order you posted. I could see the argument for loader before game versions, as game versions can be pretty long and push the loader. Don't think that's a big deal either way though

**Comment by HyperSoop** [comment]
The separation brings in too much contrast in different areas of a single version, it just doesn't fit, there isn't that much information in any one of the three.

The buttons on top are atrocious, thinking about it. "Back to list" should be a clickable text in the top right corner, "Delete" should be moved into "Edit", the remaining 3 buttons (looking just at Minenash's screenshot) should be ordered as Download-Edit-Package and never ever have line breaks.

By the way, the header seems to have way too much empty offset space above it, a whole other line could fit in there.
